locale/pt-br/texts.po

```diff
@@ -959,6 +959,14 @@ msgstr "O endereço informado não pertence a esta carteira."
 msgid "Error while trying to get Nano Contract state."
 msgstr "Erro ao obter o estado do Nano Contract."
 
+#: src/sagas/nanoContract.js:35
+msgid "Nano Contract not registered."
+msgstr "Nano Contrato não registrado."
```

Comment:
suggestion: I think we don't need to translate "Nano Contract" to "Nano Contrato", sounds weird to me, but that's just my opinion

Reply:
@pedroferreira1, should I translate it or not? We use it as a brand?

Reply:
We can keep it as Nano Contract for now

Reply:
Changed to `Nano Contract`